# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/perl/no-make-depend.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- perl-5.38.0/Configure.vanilla	2023-09-08 19:02:31.086740424 +0200
+++ perl-5.38.0/Configure	2023-09-08 19:02:45.364739486 +0200
@@ -25882,7 +25882,7 @@
 cd "$pwd"
 
 if $contains '^depend:' [Mm]akefile >/dev/null 2>&1; then
-	dflt=y
+	dflt=n
 	case "$silent" in
 	true) ;;
 	*)
